kv/kvnemesis: TestKVNemesisMultiNode failed

[(kv/kvnemesis).TestKVNemesisMultiNode failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2605102&tab=buildLog) on [master@baf34b241c9ee27530f0a3a988149bfd7cca3f6d](https://github.com/cockroachdb/cockroach/commits/baf34b241c9ee27530f0a3a988149bfd7cca3f6d):

```
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).ChangeReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_command.go:964
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeAdminBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:584
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:100
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:36
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:195
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:177
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:874
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:326
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:862
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:900
          | github.com/cockroachdb/cockroach/pkg/roachpb._Internal_Batch_Handler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/roachpb/api.pb.go:9080
          | github.com/cockroachdb/cockroach/pkg/util/tracing.ServerInterceptor.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/tracing/grpc_interceptor.go:135
          | google.golang.org/grpc.getChainUnaryHandler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:921
          | github.com/cockroachdb/cockroach/pkg/rpc.NewServer.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:176
          | google.golang.org/grpc.getChainUnaryHandler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:921
          | github.com/cockroachdb/cockroach/pkg/rpc.kvAuth.unaryInterceptor
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/auth.go:60
          | google.golang.org/grpc.chainUnaryServerInterceptors.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:907
          | github.com/cockroachdb/cockroach/pkg/roachpb._Internal_Batch_Handler
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/roachpb/api.pb.go:9082
          | google.golang.org/grpc.(*Server).processUnaryRPC
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:1082
          | google.golang.org/grpc.(*Server).handleStream
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:1405
          | google.golang.org/grpc.(*Server).serveStreams.func1.1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:746
          | runtime.goexit
          | 	/usr/local/go/src/runtime/asm_amd64.s:1374
        Wraps: (4) change replicas of r36 failed
        Wraps: (5) unexpected value: raw_bytes:"\025)ER\003\010&\022\014\272\022bc011205\000\001\032\002\377\377\"\006\010\001\020\001\030\001(\0020\003:\020\010\377\377\377\377\377\377\377\377\177\020\377\377\377\377\007" timestamp:<wall_time:1611298222189295321 > 
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *errbase.opaqueWrapper (4) *errutil.withPrefix (5) *roachpb.ConditionFailedError
    kvnemesis_test.go:84: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestKVNemesisMultiNode440379575
--- FAIL: TestKVNemesisMultiNode (0.64s)

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestKVNemesisMultiNode PKG=./pkg/kv/kvnemesis TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestKVNemesisMultiNode.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(kv/kvnemesis).TestKVNemesisMultiNode failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2622998&tab=buildLog) on [master@1a459a81dba35b6a091f0a2954aa33d50f1e5d24](https://github.com/cockroachdb/cockroach/commits/1a459a81dba35b6a091f0a2954aa33d50f1e5d24):

```
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).ChangeReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_command.go:964
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeAdminBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:561
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:99
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:35
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:195
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:177
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:875
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:331
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:863
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:901
          | github.com/cockroachdb/cockroach/pkg/roachpb._Internal_Batch_Handler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/roachpb/api.pb.go:9061
          | github.com/cockroachdb/cockroach/pkg/util/tracing.ServerInterceptor.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/tracing/grpc_interceptor.go:135
          | google.golang.org/grpc.getChainUnaryHandler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:921
          | github.com/cockroachdb/cockroach/pkg/rpc.NewServer.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:176
          | google.golang.org/grpc.getChainUnaryHandler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:921
          | github.com/cockroachdb/cockroach/pkg/rpc.kvAuth.unaryInterceptor
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/auth.go:60
          | google.golang.org/grpc.chainUnaryServerInterceptors.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:907
          | github.com/cockroachdb/cockroach/pkg/roachpb._Internal_Batch_Handler
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/roachpb/api.pb.go:9063
          | google.golang.org/grpc.(*Server).processUnaryRPC
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:1082
          | google.golang.org/grpc.(*Server).handleStream
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:1405
          | google.golang.org/grpc.(*Server).serveStreams.func1.1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:746
          | runtime.goexit
          | 	/usr/local/go/src/runtime/asm_amd64.s:1374
        Wraps: (4) change replicas of r36 failed
        Wraps: (5) unexpected value: raw_bytes:"\257\251\334\002\003\010&\022\014\272\0228c8ca801\000\001\032\002\377\377\"\006\010\001\020\001\030\001(\0020\003:\020\010\377\377\377\377\377\377\377\377\177\020\377\377\377\377\007" timestamp:<wall_time:1611817372341762141 > 
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *errbase.opaqueWrapper (4) *errutil.withPrefix (5) *roachpb.ConditionFailedError
    kvnemesis_test.go:84: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestKVNemesisMultiNode889660737
--- FAIL: TestKVNemesisMultiNode (0.61s)

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestKVNemesisMultiNode PKG=./pkg/kv/kvnemesis TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestKVNemesisMultiNode.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(kv/kvnemesis).TestKVNemesisMultiNode failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2631786&tab=buildLog) on [master@e4ac407b14ed315662050f09670707cdf8ad9bfe](https://github.com/cockroachdb/cockroach/commits/e4ac407b14ed315662050f09670707cdf8ad9bfe):

```
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).ChangeReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_command.go:969
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeAdminBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:546
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:99
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:35
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:195
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:177
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:875
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:331
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:863
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:901
          | github.com/cockroachdb/cockroach/pkg/roachpb._Internal_Batch_Handler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/roachpb/api.pb.go:9067
          | github.com/cockroachdb/cockroach/pkg/util/tracing.ServerInterceptor.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/tracing/grpc_interceptor.go:135
          | google.golang.org/grpc.getChainUnaryHandler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:921
          | github.com/cockroachdb/cockroach/pkg/rpc.NewServer.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:176
          | google.golang.org/grpc.getChainUnaryHandler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:921
          | github.com/cockroachdb/cockroach/pkg/rpc.kvAuth.unaryInterceptor
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/auth.go:60
          | google.golang.org/grpc.chainUnaryServerInterceptors.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:907
          | github.com/cockroachdb/cockroach/pkg/roachpb._Internal_Batch_Handler
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/roachpb/api.pb.go:9069
          | google.golang.org/grpc.(*Server).processUnaryRPC
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:1082
          | google.golang.org/grpc.(*Server).handleStream
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:1405
          | google.golang.org/grpc.(*Server).serveStreams.func1.1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:746
          | runtime.goexit
          | 	/usr/local/go/src/runtime/asm_amd64.s:1374
        Wraps: (4) change replicas of r36 failed
        Wraps: (5) unexpected value: raw_bytes:"Su\016D\003\010)\022\014\272\0227ea6c3de\000\001\032\002\377\377\"\006\010\001\020\001\030\001(\0020\004:\020\010\377\377\377\377\377\377\377\377\177\020\377\377\377\377\007" timestamp:<wall_time:1612161479188325274 > 
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *errbase.opaqueWrapper (4) *errutil.withPrefix (5) *roachpb.ConditionFailedError
    kvnemesis_test.go:84: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestKVNemesisMultiNode064252247
--- FAIL: TestKVNemesisMultiNode (0.74s)

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestKVNemesisMultiNode PKG=./pkg/kv/kvnemesis TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestKVNemesisMultiNode.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(kv/kvnemesis).TestKVNemesisMultiNode failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2635962&tab=buildLog) on [master@0e6727832d58faf0f900601cd6fa6807e0a2ba75](https://github.com/cockroachdb/cockroach/commits/0e6727832d58faf0f900601cd6fa6807e0a2ba75):

```
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:328
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:863
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:901
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:465
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:193
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:175
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1878
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1503
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1141
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:780
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:219
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:808
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:791
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:719
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).Run
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:742
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).AdminChangeReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:647
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.applyOp
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/applier.go:82
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*Applier).Apply
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/applier.go:58
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.RunNemesis.func2
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/kvnemesis.go:64
          | github.com/cockroachdb/cockroach/pkg/util/ctxgroup.GroupWorkers.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/ctxgroup/ctxgroup.go:175
          | github.com/cockroachdb/cockroach/pkg/util/ctxgroup.Group.GoCtx.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/ctxgroup/ctxgroup.go:166
          | golang.org/x/sync/errgroup.(*Group).Go.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup/errgroup.go:57
        Wraps: (4) change replicas of r36 failed
        Wraps: (5) unexpected value: raw_bytes:":q\321\352\003\010'\022\014\272\0223651a302\000\001\032\002\377\377\"\006\010\001\020\001\030\001(\0020\003:\020\010\377\377\377\377\377\377\377\377\177\020\377\377\377\377\007" timestamp:<wall_time:1612249859802413866 > 
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *errbase.opaqueWrapper (4) *errutil.withPrefix (5) *roachpb.ConditionFailedError
    kvnemesis_test.go:84: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestKVNemesisMultiNode972494689
--- FAIL: TestKVNemesisMultiNode (0.65s)

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestKVNemesisMultiNode PKG=./pkg/kv/kvnemesis TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestKVNemesisMultiNode.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(kv/kvnemesis).TestKVNemesisMultiNode failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2639578&tab=buildLog) on [master@4736b8dd2c36b58fffbb0a665690e2c0e719d7c3](https://github.com/cockroachdb/cockroach/commits/4736b8dd2c36b58fffbb0a665690e2c0e719d7c3):

```
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).ChangeReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_command.go:969
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeAdminBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:546
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:99
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:35
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:195
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:177
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:875
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:328
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:863
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:901
          | github.com/cockroachdb/cockroach/pkg/roachpb._Internal_Batch_Handler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/roachpb/api.pb.go:9067
          | github.com/cockroachdb/cockroach/pkg/util/tracing.ServerInterceptor.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/tracing/grpc_interceptor.go:135
          | google.golang.org/grpc.getChainUnaryHandler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:921
          | github.com/cockroachdb/cockroach/pkg/rpc.NewServer.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:176
          | google.golang.org/grpc.getChainUnaryHandler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:921
          | github.com/cockroachdb/cockroach/pkg/rpc.kvAuth.unaryInterceptor
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/auth.go:60
          | google.golang.org/grpc.chainUnaryServerInterceptors.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:907
          | github.com/cockroachdb/cockroach/pkg/roachpb._Internal_Batch_Handler
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/roachpb/api.pb.go:9069
          | google.golang.org/grpc.(*Server).processUnaryRPC
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:1082
          | google.golang.org/grpc.(*Server).handleStream
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:1405
          | google.golang.org/grpc.(*Server).serveStreams.func1.1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:746
          | runtime.goexit
          | 	/usr/local/go/src/runtime/asm_amd64.s:1374
        Wraps: (4) change replicas of r36 failed
        Wraps: (5) unexpected value: raw_bytes:"\372\2771#\003\010'\022\014\272\0228131dd00\000\001\032\002\377\377\"\006\010\001\020\001\030\001(\0020\003:\020\010\377\377\377\377\377\377\377\377\177\020\377\377\377\377\007" timestamp:<wall_time:1612334711373574852 > 
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *errbase.opaqueWrapper (4) *errutil.withPrefix (5) *roachpb.ConditionFailedError
    kvnemesis_test.go:84: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestKVNemesisMultiNode904280569
--- FAIL: TestKVNemesisMultiNode (0.62s)

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestKVNemesisMultiNode PKG=./pkg/kv/kvnemesis TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestKVNemesisMultiNode.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(kv/kvnemesis).TestKVNemesisMultiNode failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2643306&tab=buildLog) on [master@100c09f4f6eb3f5b18a67ec4bbfdfe989e0d6ce2](https://github.com/cockroachdb/cockroach/commits/100c09f4f6eb3f5b18a67ec4bbfdfe989e0d6ce2):

```
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).ChangeReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_command.go:969
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeAdminBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:546
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:99
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:35
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:195
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:177
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:875
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:331
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:863
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:901
          | github.com/cockroachdb/cockroach/pkg/roachpb._Internal_Batch_Handler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/roachpb/api.pb.go:9067
          | github.com/cockroachdb/cockroach/pkg/util/tracing.ServerInterceptor.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/tracing/grpc_interceptor.go:135
          | google.golang.org/grpc.getChainUnaryHandler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:921
          | github.com/cockroachdb/cockroach/pkg/rpc.NewServer.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:176
          | google.golang.org/grpc.getChainUnaryHandler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:921
          | github.com/cockroachdb/cockroach/pkg/rpc.kvAuth.unaryInterceptor
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/auth.go:60
          | google.golang.org/grpc.chainUnaryServerInterceptors.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:907
          | github.com/cockroachdb/cockroach/pkg/roachpb._Internal_Batch_Handler
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/roachpb/api.pb.go:9069
          | google.golang.org/grpc.(*Server).processUnaryRPC
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:1082
          | google.golang.org/grpc.(*Server).handleStream
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:1405
          | google.golang.org/grpc.(*Server).serveStreams.func1.1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:746
          | runtime.goexit
          | 	/usr/local/go/src/runtime/asm_amd64.s:1374
        Wraps: (4) change replicas of r38 failed
        Wraps: (5) unexpected value: raw_bytes:"\202\370/\302\003\010'\022\014\272\022434049b0\000\001\032\002\377\377\"\006\010\001\020\001\030\001(\0020\004:\020\010\377\377\377\377\377\377\377\377\177\020\377\377\377\377\007" timestamp:<wall_time:1612421168623291882 > 
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *errbase.opaqueWrapper (4) *errutil.withPrefix (5) *roachpb.ConditionFailedError
    kvnemesis_test.go:84: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestKVNemesisMultiNode288732811
--- FAIL: TestKVNemesisMultiNode (0.65s)

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestKVNemesisMultiNode PKG=./pkg/kv/kvnemesis TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestKVNemesisMultiNode.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(kv/kvnemesis).TestKVNemesisMultiNode failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2647049&tab=buildLog) on [master@c584f62067a45aa540c26fc9081a83e460bfe37a](https://github.com/cockroachdb/cockroach/commits/c584f62067a45aa540c26fc9081a83e460bfe37a):

```
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:331
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:863
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:901
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:465
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:193
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:175
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1878
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1503
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1141
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:780
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:219
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:808
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:791
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:719
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).Run
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:742
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).AdminChangeReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:647
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.applyOp
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/applier.go:82
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*Applier).Apply
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/applier.go:58
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.RunNemesis.func2
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/kvnemesis.go:64
          | github.com/cockroachdb/cockroach/pkg/util/ctxgroup.GroupWorkers.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/ctxgroup/ctxgroup.go:175
          | github.com/cockroachdb/cockroach/pkg/util/ctxgroup.Group.GoCtx.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/ctxgroup/ctxgroup.go:166
          | golang.org/x/sync/errgroup.(*Group).Go.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup/errgroup.go:57
        Wraps: (4) change replicas of r36 failed
        Wraps: (5) unexpected value: raw_bytes:"\304b\377\326\003\010&\022\014\272\022399cdc13\000\001\032\002\377\377\"\006\010\001\020\001\030\001(\0020\003:\020\010\377\377\377\377\377\377\377\377\177\020\377\377\377\377\007" timestamp:<wall_time:1612507572826373898 > 
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *errbase.opaqueWrapper (4) *errutil.withPrefix (5) *roachpb.ConditionFailedError
    kvnemesis_test.go:84: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestKVNemesisMultiNode901979378
--- FAIL: TestKVNemesisMultiNode (0.63s)

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestKVNemesisMultiNode PKG=./pkg/kv/kvnemesis TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestKVNemesisMultiNode.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(kv/kvnemesis).TestKVNemesisMultiNode failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2650594&tab=buildLog) on [master@81a2c26a104fa8cc7e8b530b837ffb6ff85ddc5a](https://github.com/cockroachdb/cockroach/commits/81a2c26a104fa8cc7e8b530b837ffb6ff85ddc5a):

```
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).ChangeReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_command.go:969
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeAdminBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:559
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:98
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:34
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:195
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:177
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:875
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:331
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:863
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:901
          | github.com/cockroachdb/cockroach/pkg/roachpb._Internal_Batch_Handler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/roachpb/api.pb.go:9028
          | github.com/cockroachdb/cockroach/pkg/util/tracing.ServerInterceptor.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/tracing/grpc_interceptor.go:135
          | google.golang.org/grpc.getChainUnaryHandler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:921
          | github.com/cockroachdb/cockroach/pkg/rpc.NewServer.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:176
          | google.golang.org/grpc.getChainUnaryHandler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:921
          | github.com/cockroachdb/cockroach/pkg/rpc.kvAuth.unaryInterceptor
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/auth.go:60
          | google.golang.org/grpc.chainUnaryServerInterceptors.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:907
          | github.com/cockroachdb/cockroach/pkg/roachpb._Internal_Batch_Handler
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/roachpb/api.pb.go:9030
          | google.golang.org/grpc.(*Server).processUnaryRPC
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:1082
          | google.golang.org/grpc.(*Server).handleStream
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:1405
          | google.golang.org/grpc.(*Server).serveStreams.func1.1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:746
          | runtime.goexit
          | 	/usr/local/go/src/runtime/asm_amd64.s:1374
        Wraps: (4) change replicas of r36 failed
        Wraps: (5) unexpected value: raw_bytes:"\363v`\260\003\010(\022\014\272\022b86d4d5d\000\001\032\002\377\377\"\006\010\001\020\001\030\001(\0020\003:\020\010\377\377\377\377\377\377\377\377\177\020\377\377\377\377\007" timestamp:<wall_time:1612593956760726495 logical:1 > 
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *errbase.opaqueWrapper (4) *errutil.withPrefix (5) *roachpb.ConditionFailedError
    kvnemesis_test.go:84: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestKVNemesisMultiNode758810121
--- FAIL: TestKVNemesisMultiNode (0.77s)

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestKVNemesisMultiNode PKG=./pkg/kv/kvnemesis TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestKVNemesisMultiNode.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(kv/kvnemesis).TestKVNemesisMultiNode failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2651955&tab=buildLog) on [master@0e98670c8fcec566937e899fdf77d2a68c702d62](https://github.com/cockroachdb/cockroach/commits/0e98670c8fcec566937e899fdf77d2a68c702d62):

```
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:331
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:863
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:901
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:465
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:193
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:175
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1882
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1503
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1141
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:780
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:219
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:808
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:791
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:719
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).Run
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:742
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).AdminChangeReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:647
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.applyOp
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/applier.go:82
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*Applier).Apply
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/applier.go:58
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.RunNemesis.func2
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/kvnemesis.go:64
          | github.com/cockroachdb/cockroach/pkg/util/ctxgroup.GroupWorkers.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/ctxgroup/ctxgroup.go:175
          | github.com/cockroachdb/cockroach/pkg/util/ctxgroup.Group.GoCtx.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/ctxgroup/ctxgroup.go:166
          | golang.org/x/sync/errgroup.(*Group).Go.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup/errgroup.go:57
        Wraps: (4) change replicas of r39 failed
        Wraps: (5) unexpected value: raw_bytes:"\177Z\016\264\003\010+\022\014\272\02203b962e7\000\001\032\014\272\022daa34533\000\001\"\006\010\001\020\001\030\001(\0020\004:\020\010\377\377\377\377\377\377\377\377\177\020\377\377\377\377\007" timestamp:<wall_time:1612681259676696729 > 
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *errbase.opaqueWrapper (4) *errutil.withPrefix (5) *roachpb.ConditionFailedError
    kvnemesis_test.go:84: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestKVNemesisMultiNode765448515
--- FAIL: TestKVNemesisMultiNode (0.67s)

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestKVNemesisMultiNode PKG=./pkg/kv/kvnemesis TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestKVNemesisMultiNode.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(kv/kvnemesis).TestKVNemesisMultiNode failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2653752&tab=buildLog) on [master@5b33e6dfc47000de831745a851e3bf9e2cf7fd95](https://github.com/cockroachdb/cockroach/commits/5b33e6dfc47000de831745a851e3bf9e2cf7fd95):

```
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).ChangeReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_command.go:969
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeAdminBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:559
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:98
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:34
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:195
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:177
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:875
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:331
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:863
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:901
          | github.com/cockroachdb/cockroach/pkg/roachpb._Internal_Batch_Handler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/roachpb/api.pb.go:9028
          | github.com/cockroachdb/cockroach/pkg/util/tracing.ServerInterceptor.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/tracing/grpc_interceptor.go:135
          | google.golang.org/grpc.getChainUnaryHandler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:921
          | github.com/cockroachdb/cockroach/pkg/rpc.NewServer.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:176
          | google.golang.org/grpc.getChainUnaryHandler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:921
          | github.com/cockroachdb/cockroach/pkg/rpc.kvAuth.unaryInterceptor
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/auth.go:60
          | google.golang.org/grpc.chainUnaryServerInterceptors.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:907
          | github.com/cockroachdb/cockroach/pkg/roachpb._Internal_Batch_Handler
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/roachpb/api.pb.go:9030
          | google.golang.org/grpc.(*Server).processUnaryRPC
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:1082
          | google.golang.org/grpc.(*Server).handleStream
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:1405
          | google.golang.org/grpc.(*Server).serveStreams.func1.1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:746
          | runtime.goexit
          | 	/usr/local/go/src/runtime/asm_amd64.s:1374
        Wraps: (4) change replicas of r36 failed
        Wraps: (5) unexpected value: raw_bytes:"M\005\337\\\003\010'\022\014\272\0227344f6b3\000\001\032\002\377\377\"\006\010\001\020\001\030\001(\0020\003:\020\010\377\377\377\377\377\377\377\377\177\020\377\377\377\377\007" timestamp:<wall_time:1612767063190257577 > 
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *errbase.opaqueWrapper (4) *errutil.withPrefix (5) *roachpb.ConditionFailedError
    kvnemesis_test.go:84: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestKVNemesisMultiNode797448110
--- FAIL: TestKVNemesisMultiNode (0.62s)

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestKVNemesisMultiNode PKG=./pkg/kv/kvnemesis TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestKVNemesisMultiNode.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(kv/kvnemesis).TestKVNemesisMultiNode failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2657497&tab=buildLog) on [master@7853fd32de8b6dea869f2a2a92dcd7506f4a8998](https://github.com/cockroachdb/cockroach/commits/7853fd32de8b6dea869f2a2a92dcd7506f4a8998):

```
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).ChangeReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_command.go:969
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeAdminBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:559
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:98
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:34
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:195
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:177
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:875
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:331
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:863
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:901
          | github.com/cockroachdb/cockroach/pkg/roachpb._Internal_Batch_Handler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/roachpb/api.pb.go:9028
          | github.com/cockroachdb/cockroach/pkg/util/tracing.ServerInterceptor.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/tracing/grpc_interceptor.go:137
          | google.golang.org/grpc.getChainUnaryHandler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:921
          | github.com/cockroachdb/cockroach/pkg/rpc.NewServer.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:176
          | google.golang.org/grpc.getChainUnaryHandler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:921
          | github.com/cockroachdb/cockroach/pkg/rpc.kvAuth.unaryInterceptor
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/auth.go:60
          | google.golang.org/grpc.chainUnaryServerInterceptors.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:907
          | github.com/cockroachdb/cockroach/pkg/roachpb._Internal_Batch_Handler
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/roachpb/api.pb.go:9030
          | google.golang.org/grpc.(*Server).processUnaryRPC
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:1082
          | google.golang.org/grpc.(*Server).handleStream
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:1405
          | google.golang.org/grpc.(*Server).serveStreams.func1.1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:746
          | runtime.goexit
          | 	/usr/local/go/src/runtime/asm_amd64.s:1374
        Wraps: (4) change replicas of r36 failed
        Wraps: (5) unexpected value: raw_bytes:"\201\201\303\310\003\010&\022\014\272\0221f4c1630\000\001\032\002\377\377\"\006\010\001\020\001\030\001(\0020\003:\020\010\377\377\377\377\377\377\377\377\177\020\377\377\377\377\007" timestamp:<wall_time:1612853735746378450 > 
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *errbase.opaqueWrapper (4) *errutil.withPrefix (5) *roachpb.ConditionFailedError
    kvnemesis_test.go:84: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestKVNemesisMultiNode951235380
--- FAIL: TestKVNemesisMultiNode (0.64s)

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestKVNemesisMultiNode PKG=./pkg/kv/kvnemesis TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestKVNemesisMultiNode.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(kv/kvnemesis).TestKVNemesisMultiNode failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2666291&tab=buildLog) on [master@88c1ebeac2ac912ccada1e43ef212683151a2efe](https://github.com/cockroachdb/cockroach/commits/88c1ebeac2ac912ccada1e43ef212683151a2efe):

```
=== RUN   TestKVNemesisMultiNode
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestKVNemesisMultiNode806944140
    test_log_scope.go:74: use -show-logs to present logs inline
    kvnemesis_test.go:82: failure:
        error applying x.AdminSplit(ctx, /Table/50/"95cf8d14") // invalid lease: invalid lease
        (1) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).failIfError
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/validator.go:617
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).processOp
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/validator.go:291
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.Validate
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/validator.go:73
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.RunNemesis
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/kvnemesis.go:100
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.TestKVNemesisMultiNode
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/kvnemesis_test.go:78
          | testing.tRunner
          | 	/usr/local/go/src/testing/testing.go:1123
          | runtime.goexit
          | 	/usr/local/go/src/runtime/asm_amd64.s:1374
        Wraps: (2) error applying x.AdminSplit(ctx, /Table/50/"95cf8d14") // invalid lease
        Wraps: (3) invalid lease
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *roachpb.InvalidLeaseError
    kvnemesis_test.go:84: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestKVNemesisMultiNode806944140
--- FAIL: TestKVNemesisMultiNode (0.70s)

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestKVNemesisMultiNode PKG=./pkg/kv/kvnemesis TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestKVNemesisMultiNode.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(kv/kvnemesis).TestKVNemesisMultiNode failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2669698&tab=buildLog) on [master@86f242a6c58cd680844bb4ca580db0246eeb33d9](https://github.com/cockroachdb/cockroach/commits/86f242a6c58cd680844bb4ca580db0246eeb33d9):

```
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:331
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:863
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:901
          | github.com/cockroachdb/cockroach/pkg/rpc.internalClientAdapter.Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:465
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).sendBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:193
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*grpcTransport).SendNext
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/transport.go:175
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendToReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1882
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).sendPartialBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1503
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).divideAndSendBatchToRanges
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:1141
          | github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord.(*DistSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvclient/kvcoord/dist_sender.go:780
          | github.com/cockroachdb/cockroach/pkg/kv.(*CrossRangeTxnWrapperSender).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:219
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).sendUsingSender
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:808
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:791
          | github.com/cockroachdb/cockroach/pkg/kv.sendAndFill
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:719
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).Run
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:742
          | github.com/cockroachdb/cockroach/pkg/kv.(*DB).AdminChangeReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/db.go:647
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.applyOp
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/applier.go:82
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*Applier).Apply
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/applier.go:58
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.RunNemesis.func2
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/kvnemesis.go:65
          | github.com/cockroachdb/cockroach/pkg/util/ctxgroup.GroupWorkers.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/ctxgroup/ctxgroup.go:175
          | github.com/cockroachdb/cockroach/pkg/util/ctxgroup.Group.GoCtx.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/ctxgroup/ctxgroup.go:166
          | golang.org/x/sync/errgroup.(*Group).Go.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/sync/errgroup/errgroup.go:57
        Wraps: (4) change replicas of r37 failed
        Wraps: (5) unexpected value: raw_bytes:"\004\335\205F\003\010*\022\014\272\022c1dda1fa\000\001\032\002\377\377\"\006\010\001\020\001\030\001(\0020\004:\020\010\377\377\377\377\377\377\377\377\177\020\377\377\377\377\007" timestamp:<wall_time:1613094754824170282 logical:1 > 
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *errbase.opaqueWrapper (4) *errutil.withPrefix (5) *roachpb.ConditionFailedError
    kvnemesis_test.go:84: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestKVNemesisMultiNode070940468
--- FAIL: TestKVNemesisMultiNode (1.09s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestKVNemesisMultiNode PKG=./pkg/kv/kvnemesis TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestKVNemesisMultiNode.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(kv/kvnemesis).TestKVNemesisMultiNode failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2670472&tab=buildLog) on [master@e9e372122a2e3db7090b5705da07128f828e2441](https://github.com/cockroachdb/cockroach/commits/e9e372122a2e3db7090b5705da07128f828e2441):

```
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).ChangeReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_command.go:969
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeAdminBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:559
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:98
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:34
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:195
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:177
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:875
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:331
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:863
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:901
          | github.com/cockroachdb/cockroach/pkg/roachpb._Internal_Batch_Handler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/roachpb/api.pb.go:9028
          | github.com/cockroachdb/cockroach/pkg/util/tracing.ServerInterceptor.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/tracing/grpc_interceptor.go:137
          | google.golang.org/grpc.getChainUnaryHandler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:921
          | github.com/cockroachdb/cockroach/pkg/rpc.NewServer.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:176
          | google.golang.org/grpc.getChainUnaryHandler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:921
          | github.com/cockroachdb/cockroach/pkg/rpc.kvAuth.unaryInterceptor
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/auth.go:60
          | google.golang.org/grpc.chainUnaryServerInterceptors.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:907
          | github.com/cockroachdb/cockroach/pkg/roachpb._Internal_Batch_Handler
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/roachpb/api.pb.go:9030
          | google.golang.org/grpc.(*Server).processUnaryRPC
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:1082
          | google.golang.org/grpc.(*Server).handleStream
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:1405
          | google.golang.org/grpc.(*Server).serveStreams.func1.1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:746
          | runtime.goexit
          | 	/usr/local/go/src/runtime/asm_amd64.s:1374
        Wraps: (4) change replicas of r38 failed
        Wraps: (5) unexpected value: raw_bytes:"\3676\"\264\003\010+\022\014\272\02264a60b67\000\001\032\014\272\022e932e22c\000\001\"\006\010\001\020\001\030\001\"\006\010\004\020\004\030\002\"\010\010\002\020\002\030\003 \001(\0040\007:\020\010\377\377\377\377\377\377\377\377\177\020\377\377\377\377\007" timestamp:<wall_time:1613113342009781181 logical:1 > 
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *errbase.opaqueWrapper (4) *errutil.withPrefix (5) *roachpb.ConditionFailedError
    kvnemesis_test.go:84: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestKVNemesisMultiNode444110117
--- FAIL: TestKVNemesisMultiNode (0.74s)

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestKVNemesisMultiNode PKG=./pkg/kv/kvnemesis TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestKVNemesisMultiNode.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(kv/kvnemesis).TestKVNemesisMultiNode failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2673768&tab=buildLog) on [master@9969862335cf501c1aa67a4aaf018ad5dc8e85e8](https://github.com/cockroachdb/cockroach/commits/9969862335cf501c1aa67a4aaf018ad5dc8e85e8):

```
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).ChangeReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_command.go:969
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeAdminBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:559
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:98
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:34
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:195
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:177
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:875
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:331
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:863
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:901
          | github.com/cockroachdb/cockroach/pkg/roachpb._Internal_Batch_Handler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/roachpb/api.pb.go:9036
          | github.com/cockroachdb/cockroach/pkg/util/tracing.ServerInterceptor.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/tracing/grpc_interceptor.go:137
          | google.golang.org/grpc.getChainUnaryHandler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:921
          | github.com/cockroachdb/cockroach/pkg/rpc.NewServer.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:176
          | google.golang.org/grpc.getChainUnaryHandler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:921
          | github.com/cockroachdb/cockroach/pkg/rpc.kvAuth.unaryInterceptor
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/auth.go:60
          | google.golang.org/grpc.chainUnaryServerInterceptors.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:907
          | github.com/cockroachdb/cockroach/pkg/roachpb._Internal_Batch_Handler
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/roachpb/api.pb.go:9038
          | google.golang.org/grpc.(*Server).processUnaryRPC
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:1082
          | google.golang.org/grpc.(*Server).handleStream
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:1405
          | google.golang.org/grpc.(*Server).serveStreams.func1.1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:746
          | runtime.goexit
          | 	/usr/local/go/src/runtime/asm_amd64.s:1374
        Wraps: (4) change replicas of r37 failed
        Wraps: (5) unexpected value: raw_bytes:"s\332\330\354\003\010'\022\014\272\022e815cdda\000\001\032\002\377\377\"\006\010\001\020\001\030\001(\0020\003:\020\010\377\377\377\377\377\377\377\377\177\020\377\377\377\377\007" timestamp:<wall_time:1613199249970107640 logical:1 > 
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *errbase.opaqueWrapper (4) *errutil.withPrefix (5) *roachpb.ConditionFailedError
    kvnemesis_test.go:84: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestKVNemesisMultiNode998612964
--- FAIL: TestKVNemesisMultiNode (0.64s)

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestKVNemesisMultiNode PKG=./pkg/kv/kvnemesis TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestKVNemesisMultiNode.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(kv/kvnemesis).TestKVNemesisMultiNode failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2675027&tab=buildLog) on [master@9969862335cf501c1aa67a4aaf018ad5dc8e85e8](https://github.com/cockroachdb/cockroach/commits/9969862335cf501c1aa67a4aaf018ad5dc8e85e8):

```
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).ChangeReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_command.go:969
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeAdminBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:559
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:98
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:34
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:195
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:177
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:875
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:331
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:863
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:901
          | github.com/cockroachdb/cockroach/pkg/roachpb._Internal_Batch_Handler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/roachpb/api.pb.go:9036
          | github.com/cockroachdb/cockroach/pkg/util/tracing.ServerInterceptor.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/tracing/grpc_interceptor.go:137
          | google.golang.org/grpc.getChainUnaryHandler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:921
          | github.com/cockroachdb/cockroach/pkg/rpc.NewServer.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:176
          | google.golang.org/grpc.getChainUnaryHandler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:921
          | github.com/cockroachdb/cockroach/pkg/rpc.kvAuth.unaryInterceptor
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/auth.go:60
          | google.golang.org/grpc.chainUnaryServerInterceptors.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:907
          | github.com/cockroachdb/cockroach/pkg/roachpb._Internal_Batch_Handler
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/roachpb/api.pb.go:9038
          | google.golang.org/grpc.(*Server).processUnaryRPC
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:1082
          | google.golang.org/grpc.(*Server).handleStream
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:1405
          | google.golang.org/grpc.(*Server).serveStreams.func1.1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:746
          | runtime.goexit
          | 	/usr/local/go/src/runtime/asm_amd64.s:1374
        Wraps: (4) change replicas of r37 failed
        Wraps: (5) unexpected value: raw_bytes:"'\3730\031\003\010'\022\014\272\022c8d286aa\000\001\032\014\272\022f9311959\000\001\"\006\010\001\020\001\030\001(\0020\004:\020\010\377\377\377\377\377\377\377\377\177\020\377\377\377\377\007" timestamp:<wall_time:1613285676531187268 > 
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *errbase.opaqueWrapper (4) *errutil.withPrefix (5) *roachpb.ConditionFailedError
    kvnemesis_test.go:84: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestKVNemesisMultiNode441055465
--- FAIL: TestKVNemesisMultiNode (0.61s)

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestKVNemesisMultiNode PKG=./pkg/kv/kvnemesis TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestKVNemesisMultiNode.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(kv/kvnemesis).TestKVNemesisMultiNode failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2676999&tab=buildLog) on [master@ac0f819feeee435af2545034bbd40113b47d5b86](https://github.com/cockroachdb/cockroach/commits/ac0f819feeee435af2545034bbd40113b47d5b86):

```
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).ChangeReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_command.go:969
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeAdminBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:559
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:98
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:34
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:195
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:177
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:875
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:331
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:863
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:901
          | github.com/cockroachdb/cockroach/pkg/roachpb._Internal_Batch_Handler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/roachpb/api.pb.go:9036
          | github.com/cockroachdb/cockroach/pkg/util/tracing.ServerInterceptor.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/tracing/grpc_interceptor.go:137
          | google.golang.org/grpc.getChainUnaryHandler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:921
          | github.com/cockroachdb/cockroach/pkg/rpc.NewServer.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:176
          | google.golang.org/grpc.getChainUnaryHandler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:921
          | github.com/cockroachdb/cockroach/pkg/rpc.kvAuth.unaryInterceptor
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/auth.go:60
          | google.golang.org/grpc.chainUnaryServerInterceptors.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:907
          | github.com/cockroachdb/cockroach/pkg/roachpb._Internal_Batch_Handler
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/roachpb/api.pb.go:9038
          | google.golang.org/grpc.(*Server).processUnaryRPC
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:1082
          | google.golang.org/grpc.(*Server).handleStream
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:1405
          | google.golang.org/grpc.(*Server).serveStreams.func1.1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:746
          | runtime.goexit
          | 	/usr/local/go/src/runtime/asm_amd64.s:1374
        Wraps: (4) change replicas of r38 failed
        Wraps: (5) unexpected value: raw_bytes:"l\325H\024\003\010'\022\014\272\0223c922d00\000\001\032\002\377\377\"\006\010\001\020\001\030\001(\0020\003:\020\010\377\377\377\377\377\377\377\377\177\020\377\377\377\377\007" timestamp:<wall_time:1613372184454187834 > 
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *errbase.opaqueWrapper (4) *errutil.withPrefix (5) *roachpb.ConditionFailedError
    kvnemesis_test.go:84: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestKVNemesisMultiNode173292767
--- FAIL: TestKVNemesisMultiNode (0.58s)

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestKVNemesisMultiNode PKG=./pkg/kv/kvnemesis TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestKVNemesisMultiNode.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(kv/kvnemesis).TestKVNemesisMultiNode failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2679532&tab=buildLog) on [master@68b86d57c7f36b9b6926d1fb82fda0cb5674a8f1](https://github.com/cockroachdb/cockroach/commits/68b86d57c7f36b9b6926d1fb82fda0cb5674a8f1):

```
=== RUN   TestKVNemesisMultiNode
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestKVNemesisMultiNode783508487
    test_log_scope.go:74: use -show-logs to present logs inline
    kvnemesis_test.go:82: failure:
        error applying x.AdminChangeReplicas(ctx, /Table/50/"f8376f62", [{ADD_VOTER n4,s4}]) // invalid lease: invalid lease
        (1) attached stack trace
          -- stack trace:
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).failIfError
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/validator.go:617
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.(*validator).processOp
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/validator.go:341
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.Validate
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/validator.go:73
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.RunNemesis
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/kvnemesis.go:100
          | github.com/cockroachdb/cockroach/pkg/kv/kvnemesis.TestKVNemesisMultiNode
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/kvnemesis_test.go:78
          | testing.tRunner
          | 	/usr/local/go/src/testing/testing.go:1123
          | runtime.goexit
          | 	/usr/local/go/src/runtime/asm_amd64.s:1374
        Wraps: (2) error applying x.AdminChangeReplicas(ctx, /Table/50/"f8376f62", [{ADD_VOTER n4,s4}]) // invalid lease
        Wraps: (3) invalid lease
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *roachpb.InvalidLeaseError
    kvnemesis_test.go:84: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestKVNemesisMultiNode783508487
--- FAIL: TestKVNemesisMultiNode (0.59s)

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestKVNemesisMultiNode PKG=./pkg/kv/kvnemesis TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestKVNemesisMultiNode.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(kv/kvnemesis).TestKVNemesisMultiNode failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2684061&tab=buildLog) on [master@8a9268122caaf36ce80be35669388767393ea7a3](https://github.com/cockroachdb/cockroach/commits/8a9268122caaf36ce80be35669388767393ea7a3):

```
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).ChangeReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_command.go:977
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeAdminBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:559
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:98
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:34
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:195
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:177
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:875
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:331
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:863
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:901
          | github.com/cockroachdb/cockroach/pkg/roachpb._Internal_Batch_Handler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/roachpb/api.pb.go:9036
          | github.com/cockroachdb/cockroach/pkg/util/tracing.ServerInterceptor.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/tracing/grpc_interceptor.go:137
          | google.golang.org/grpc.getChainUnaryHandler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:921
          | github.com/cockroachdb/cockroach/pkg/rpc.NewServer.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:176
          | google.golang.org/grpc.getChainUnaryHandler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:921
          | github.com/cockroachdb/cockroach/pkg/rpc.kvAuth.unaryInterceptor
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/auth.go:60
          | google.golang.org/grpc.chainUnaryServerInterceptors.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:907
          | github.com/cockroachdb/cockroach/pkg/roachpb._Internal_Batch_Handler
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/roachpb/api.pb.go:9038
          | google.golang.org/grpc.(*Server).processUnaryRPC
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:1082
          | google.golang.org/grpc.(*Server).handleStream
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:1405
          | google.golang.org/grpc.(*Server).serveStreams.func1.1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:746
          | runtime.goexit
          | 	/usr/local/go/src/runtime/asm_amd64.s:1374
        Wraps: (4) change replicas of r37 failed
        Wraps: (5) unexpected value: raw_bytes:"\366\372!\007\003\010&\022\014\272\02278a71fc0\000\001\032\002\377\377\"\006\010\001\020\001\030\001(\0020\003:\020\010\377\377\377\377\377\377\377\377\177\020\377\377\377\377\007" timestamp:<wall_time:1613545058807462921 > 
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *errbase.opaqueWrapper (4) *errutil.withPrefix (5) *roachpb.ConditionFailedError
    kvnemesis_test.go:84: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestKVNemesisMultiNode266764770
--- FAIL: TestKVNemesisMultiNode (0.60s)

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestKVNemesisMultiNode PKG=./pkg/kv/kvnemesis TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestKVNemesisMultiNode.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(kv/kvnemesis).TestKVNemesisMultiNode failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2688807&tab=buildLog) on [master@86a44f733238b8999d1abe316aa48d3a3ccf5fc3](https://github.com/cockroachdb/cockroach/commits/86a44f733238b8999d1abe316aa48d3a3ccf5fc3):

Fatal error:
```
panic: runtime error: index out of range [0] with length 0 [recovered]
	panic: runtime error: index out of range [0] with length 0

```

Stack:
```
goroutine 3031 [running]:
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Recover(0xc000ddc300, 0x51656e0, 0xc001c713b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:233 +0x126
panic(0x410fd60, 0xc004c0c600)
	/usr/local/go/src/runtime/panic.go:969 +0x1b9
golang.org/x/net/trace.(*trace).addEvent(0xc002f59040, 0x3d0f2a0, 0xc004a6ced0, 0xc004a60000)
	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/net/trace/trace.go:805 +0x71d
golang.org/x/net/trace.(*trace).LazyPrintf(0xc002f59040, 0x4351c77, 0x2, 0xc004c12020, 0x1, 0x1)
	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/net/trace/trace.go:834 +0x9a
github.com/cockroachdb/cockroach/pkg/util/tracing.(*Span).Recordf(0xc004de6280, 0x4351c77, 0x2, 0xc004c12000, 0x1, 0x1)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/tracing/span.go:367 +0x1ba
github.com/cockroachdb/cockroach/pkg/util/tracing.(*Span).Record(...)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/tracing/span.go:354
github.com/cockroachdb/cockroach/pkg/util/log.eventInternal(0xc004de6280, 0x0, 0x0, 0x1, 0x1664c48b4a2f6500, 0xbd7, 0x6dc227d, 0x1b, 0x41a, 0xc0041ad580, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/trace.go:148 +0x778
github.com/cockroachdb/cockroach/pkg/util/log.Eventf(0x5184d60, 0xc004de63a0, 0x43c5370, 0x24, 0xc002e99190, 0x2, 0x2)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/trace.go:228 +0x158
github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).refreshProposalsLocked(0xc004b9a000, 0x51656e0, 0xc0041e6f90, 0x0, 0x1)
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:1050 +0x868
github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).handleRaftReadyRaftMuLocked(0xc004b9a000, 0x51656e0, 0xc0041e6f90, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:816 +0x1006
github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).handleRaftReady(0xc004b9a000, 0x51656e0, 0xc0041e6f90, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_raft.go:445 +0x110
github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).processReady(0xc001cf2000, 0x51656e0, 0xc001c713b0, 0x27)
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_raft.go:523 +0x134
github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*raftScheduler).worker(0xc001b7c140, 0x51656e0, 0xc001c713b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/scheduler.go:284 +0x2c2
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask.func1(0xc000ddc300, 0x51656e0, 0xc001c713b0, 0x0, 0xc001ce7d90)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:351 +0xb9
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTask
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:346 +0xfc

```

<details><summary>Log preceding fatal error</summary><p>

```
=== RUN   TestKVNemesisMultiNode
    test_log_scope.go:73: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestKVNemesisMultiNode310968461
    test_log_scope.go:74: use -show-logs to present logs inline

```

</p></details>

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestKVNemesisMultiNode PKG=./pkg/kv/kvnemesis TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestKVNemesisMultiNode.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(kv/kvnemesis).TestKVNemesisMultiNode failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2693604&tab=buildLog) on [master@83e70ce84b740e27e721c3b73c38a4b8b515094a](https://github.com/cockroachdb/cockroach/commits/83e70ce84b740e27e721c3b73c38a4b8b515094a):

```
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).ChangeReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_command.go:977
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeAdminBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:559
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:98
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:34
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:195
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:177
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:875
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:330
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:863
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:901
          | github.com/cockroachdb/cockroach/pkg/roachpb._Internal_Batch_Handler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/roachpb/api.pb.go:9036
          | github.com/cockroachdb/cockroach/pkg/util/tracing.ServerInterceptor.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/tracing/grpc_interceptor.go:137
          | google.golang.org/grpc.getChainUnaryHandler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:921
          | github.com/cockroachdb/cockroach/pkg/rpc.NewServer.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:176
          | google.golang.org/grpc.getChainUnaryHandler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:921
          | github.com/cockroachdb/cockroach/pkg/rpc.kvAuth.unaryInterceptor
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/auth.go:60
          | google.golang.org/grpc.chainUnaryServerInterceptors.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:907
          | github.com/cockroachdb/cockroach/pkg/roachpb._Internal_Batch_Handler
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/roachpb/api.pb.go:9038
          | google.golang.org/grpc.(*Server).processUnaryRPC
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:1082
          | google.golang.org/grpc.(*Server).handleStream
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:1405
          | google.golang.org/grpc.(*Server).serveStreams.func1.1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:746
          | runtime.goexit
          | 	/usr/local/go/src/runtime/asm_amd64.s:1374
        Wraps: (4) change replicas of r37 failed
        Wraps: (5) unexpected value: raw_bytes:"\346\203u*\003\010'\022\014\272\022dddc02bb\000\001\032\002\377\377\"\006\010\001\020\001\030\001(\0020\003:\020\010\377\377\377\377\377\377\377\377\177\020\377\377\377\377\007" timestamp:<wall_time:1613717207698981886 > 
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *errbase.opaqueWrapper (4) *errutil.withPrefix (5) *roachpb.ConditionFailedError
    kvnemesis_test.go:87: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestKVNemesisMultiNode566059966
--- FAIL: TestKVNemesisMultiNode (0.65s)

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestKVNemesisMultiNode PKG=./pkg/kv/kvnemesis TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestKVNemesisMultiNode.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(kv/kvnemesis).TestKVNemesisMultiNode failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2697765&tab=buildLog) on [master@8b6f3c84cc256debeeb4d4055c5f0d5c9a481213](https://github.com/cockroachdb/cockroach/commits/8b6f3c84cc256debeeb4d4055c5f0d5c9a481213):

```
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).ChangeReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_command.go:978
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeAdminBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:559
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:98
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:34
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:195
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:177
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:875
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:330
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:863
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:901
          | github.com/cockroachdb/cockroach/pkg/roachpb._Internal_Batch_Handler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/roachpb/api.pb.go:9050
          | github.com/cockroachdb/cockroach/pkg/util/tracing.ServerInterceptor.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/tracing/grpc_interceptor.go:137
          | google.golang.org/grpc.getChainUnaryHandler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:921
          | github.com/cockroachdb/cockroach/pkg/rpc.NewServer.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:176
          | google.golang.org/grpc.getChainUnaryHandler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:921
          | github.com/cockroachdb/cockroach/pkg/rpc.kvAuth.unaryInterceptor
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/auth.go:60
          | google.golang.org/grpc.chainUnaryServerInterceptors.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:907
          | github.com/cockroachdb/cockroach/pkg/roachpb._Internal_Batch_Handler
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/roachpb/api.pb.go:9052
          | google.golang.org/grpc.(*Server).processUnaryRPC
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:1082
          | google.golang.org/grpc.(*Server).handleStream
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:1405
          | google.golang.org/grpc.(*Server).serveStreams.func1.1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:746
          | runtime.goexit
          | 	/usr/local/go/src/runtime/asm_amd64.s:1374
        Wraps: (4) change replicas of r37 failed
        Wraps: (5) unexpected value: raw_bytes:"3+\276^\003\010)\022\014\272\0224ca3c8ce\000\001\032\002\377\377\"\006\010\001\020\001\030\001(\0020\004:\020\010\377\377\377\377\377\377\377\377\177\020\377\377\377\377\007" timestamp:<wall_time:1613804036246666456 > 
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *errbase.opaqueWrapper (4) *errutil.withPrefix (5) *roachpb.ConditionFailedError
    kvnemesis_test.go:87: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestKVNemesisMultiNode101050788
--- FAIL: TestKVNemesisMultiNode (0.67s)

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestKVNemesisMultiNode PKG=./pkg/kv/kvnemesis TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestKVNemesisMultiNode.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(kv/kvnemesis).TestKVNemesisMultiNode failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2699729&tab=buildLog) on [master@64c4aef909f4382523cd9248341ca9f4448d841a](https://github.com/cockroachdb/cockroach/commits/64c4aef909f4382523cd9248341ca9f4448d841a):

```
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).ChangeReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_command.go:978
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeAdminBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:559
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:98
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:34
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:195
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:177
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:875
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:330
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:863
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:901
          | github.com/cockroachdb/cockroach/pkg/roachpb._Internal_Batch_Handler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/roachpb/api.pb.go:9050
          | github.com/cockroachdb/cockroach/pkg/util/tracing.ServerInterceptor.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/tracing/grpc_interceptor.go:137
          | google.golang.org/grpc.getChainUnaryHandler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:921
          | github.com/cockroachdb/cockroach/pkg/rpc.NewServer.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:176
          | google.golang.org/grpc.getChainUnaryHandler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:921
          | github.com/cockroachdb/cockroach/pkg/rpc.kvAuth.unaryInterceptor
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/auth.go:60
          | google.golang.org/grpc.chainUnaryServerInterceptors.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:907
          | github.com/cockroachdb/cockroach/pkg/roachpb._Internal_Batch_Handler
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/roachpb/api.pb.go:9052
          | google.golang.org/grpc.(*Server).processUnaryRPC
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:1082
          | google.golang.org/grpc.(*Server).handleStream
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:1405
          | google.golang.org/grpc.(*Server).serveStreams.func1.1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:746
          | runtime.goexit
          | 	/usr/local/go/src/runtime/asm_amd64.s:1374
        Wraps: (4) change replicas of r38 failed
        Wraps: (5) unexpected value: raw_bytes:"g\231<\316\003\010'\022\014\272\0226f2649f6\000\001\032\002\377\377\"\006\010\001\020\001\030\001(\0020\003:\020\010\377\377\377\377\377\377\377\377\177\020\377\377\377\377\007" timestamp:<wall_time:1613892014402992708 > 
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *errbase.opaqueWrapper (4) *errutil.withPrefix (5) *roachpb.ConditionFailedError
    kvnemesis_test.go:87: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestKVNemesisMultiNode639316699
--- FAIL: TestKVNemesisMultiNode (0.71s)

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestKVNemesisMultiNode PKG=./pkg/kv/kvnemesis TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestKVNemesisMultiNode.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(kv/kvnemesis).TestKVNemesisMultiNode failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2702599&tab=buildLog) on [master@bf9744bad5a416a4b06907f0f3dd42896f7342f3](https://github.com/cockroachdb/cockroach/commits/bf9744bad5a416a4b06907f0f3dd42896f7342f3):

```
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).ChangeReplicas
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_command.go:978
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).executeAdminBatch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:559
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).sendWithRangeID
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:98
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Replica).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/replica_send.go:34
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Store).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/store_send.go:195
          | github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*Stores).Send
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/kv/kvserver/stores.go:177
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:875
          | github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:330
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:863
          | github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:901
          | github.com/cockroachdb/cockroach/pkg/roachpb._Internal_Batch_Handler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/roachpb/api.pb.go:9050
          | github.com/cockroachdb/cockroach/pkg/util/tracing.ServerInterceptor.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/util/tracing/grpc_interceptor.go:137
          | google.golang.org/grpc.getChainUnaryHandler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:921
          | github.com/cockroachdb/cockroach/pkg/rpc.NewServer.func1
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:176
          | google.golang.org/grpc.getChainUnaryHandler.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:921
          | github.com/cockroachdb/cockroach/pkg/rpc.kvAuth.unaryInterceptor
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/auth.go:60
          | google.golang.org/grpc.chainUnaryServerInterceptors.func1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:907
          | github.com/cockroachdb/cockroach/pkg/roachpb._Internal_Batch_Handler
          | 	/go/src/github.com/cockroachdb/cockroach/pkg/roachpb/api.pb.go:9052
          | google.golang.org/grpc.(*Server).processUnaryRPC
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:1082
          | google.golang.org/grpc.(*Server).handleStream
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:1405
          | google.golang.org/grpc.(*Server).serveStreams.func1.1
          | 	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:746
          | runtime.goexit
          | 	/usr/local/go/src/runtime/asm_amd64.s:1374
        Wraps: (4) change replicas of r37 failed
        Wraps: (5) unexpected value: raw_bytes:"}\265\372\375\003\010'\022\014\272\0222e092315\000\001\032\002\377\377\"\006\010\001\020\001\030\001(\0020\003:\020\010\377\377\377\377\377\377\377\377\177\020\377\377\377\377\007" timestamp:<wall_time:1613978002654090734 > 
        Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *errbase.opaqueWrapper (4) *errutil.withPrefix (5) *roachpb.ConditionFailedError
    kvnemesis_test.go:87: -- test log scope end --
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestKVNemesisMultiNode542534964
--- FAIL: TestKVNemesisMultiNode (0.74s)

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestKVNemesisMultiNode PKG=./pkg/kv/kvnemesis TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestKVNemesisMultiNode.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/kv/kvserver/client_replica_test.go
+++ b/pkg/kv/kvserver/client_replica_test.go
@@ -2945,13 +2945,13 @@ func TestChangeReplicasLeaveAtomicRacesWithMerge(t *testing.T) {
 				ctx, rhs, *rhsDesc, roachpb.MakeReplicationChanges(roachpb.ADD_VOTER, tc.Target(2)),
 			)
 			// We'll ultimately fail because we're going to race with the work below.
-			msg := "descriptor changed"
+			msg := `descriptor changed:`
 			if resplit {
-				// We don't convert ConditionFailedError to the "descriptor changed"
-				// error if the range ID changed.
-				msg = "unexpected value"
+				// We return a more detailed "descriptor changed" error if the
+				// range ID changed.
+				msg = `descriptor changed: .* \(range replaced\)`
 			}
-			require.True(t, testutils.IsError(err, msg), err)
+			require.Regexp(t, msg, err)
 		}()
 		// Wait until our goroutine is blocked.
 		testutils.SucceedsSoon(t, func() error {

--- a/pkg/kv/kvserver/markers.go
+++ b/pkg/kv/kvserver/markers.go
@@ -36,22 +36,33 @@ func IsRetriableReplicationChangeError(err error) bool {
 }
 
 const (
-	descChangedRangeSubsumedErrorFmt = "descriptor changed: expected %s != [actual] nil (range subsumed)"
 	descChangedErrorFmt              = "descriptor changed: [expected] %s != [actual] %s"
+	descChangedRangeReplacedErrorFmt = "descriptor changed: [expected] %s != [actual] %s  (range replaced)"
+	descChangedRangeSubsumedErrorFmt = "descriptor changed: [expected] %s != [actual] nil (range subsumed)"
 )
 
 func newDescChangedError(desc, actualDesc *roachpb.RangeDescriptor) error {
+	var err error
 	if actualDesc == nil {
-		return errors.Mark(errors.Newf(descChangedRangeSubsumedErrorFmt, desc), errMarkCanRetryReplicationChangeWithUpdatedDesc)
+		err = errors.Newf(descChangedRangeSubsumedErrorFmt, desc)
+	} else if desc.RangeID != actualDesc.RangeID {
+		err = errors.Newf(descChangedRangeReplacedErrorFmt, desc, actualDesc)
+	} else {
+		err = errors.Newf(descChangedErrorFmt, desc, actualDesc)
 	}
-	return errors.Mark(errors.Newf(descChangedErrorFmt, desc, actualDesc), errMarkCanRetryReplicationChangeWithUpdatedDesc)
+	return errors.Mark(err, errMarkCanRetryReplicationChangeWithUpdatedDesc)
 }
 
 func wrapDescChangedError(err error, desc, actualDesc *roachpb.RangeDescriptor) error {
+	var wrapped error
 	if actualDesc == nil {
-		return errors.Mark(errors.Wrapf(err, descChangedRangeSubsumedErrorFmt, desc), errMarkCanRetryReplicationChangeWithUpdatedDesc)
+		wrapped = errors.Wrapf(err, descChangedRangeSubsumedErrorFmt, desc)
+	} else if desc.RangeID != actualDesc.RangeID {
+		wrapped = errors.Wrapf(err, descChangedRangeReplacedErrorFmt, desc, actualDesc)
+	} else {
+		wrapped = errors.Wrapf(err, descChangedErrorFmt, desc, actualDesc)
 	}
-	return errors.Mark(errors.Wrapf(err, descChangedErrorFmt, desc, actualDesc), errMarkCanRetryReplicationChangeWithUpdatedDesc)
+	return errors.Mark(wrapped, errMarkCanRetryReplicationChangeWithUpdatedDesc)
 }
 
 // NB: don't change the string here; this will cause cross-version issues

--- a/pkg/kv/kvserver/replica_command.go
+++ b/pkg/kv/kvserver/replica_command.go
@@ -71,8 +71,7 @@ func maybeDescriptorChangedError(
 		var actualDesc roachpb.RangeDescriptor
 		if !detail.ActualValue.IsPresent() {
 			return true, nil
-		} else if err := detail.ActualValue.GetProto(&actualDesc); err == nil &&
-			desc.RangeID == actualDesc.RangeID && !desc.Equal(&actualDesc) {
+		} else if err := detail.ActualValue.GetProto(&actualDesc); err == nil && !desc.Equal(&actualDesc) {
 			return true, &actualDesc
 		}
 	}

